### DISCUSSION TITLE
Migrate to PyWavelet for CWT

### DISCUSSION THREAD

**Discussion by igor-pechersky** [blue-yonder/tsfresh#1096] [General]
https://github.com/blue-yonder/tsfresh/blob/main/tsfresh/feature_extraction/feature_calculators.py#L1356 uses `scipy.signal.cwt`
However, that method was deprecated more the year ago, and will be removed from the next SciPy release
https://github.com/scipy/scipy/commit/7819f1bd16d42999b4d5c97e0ee574bb5c925d9e

Could we migrate to PyWavelet?

**Comment by igor-pechersky** [comment]
https://github.com/blue-yonder/tsfresh/pull/1097

**Reply by nils-braun** [reply]
Thank you very much @igor-pechersky!
